src/exception/exception-api.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
```

Comment:
Maybe we just want to do a repo-wide find and update for 2023

Reply:
https://github.com/icgc-argo/argo-clinical/pull/947